### PR TITLE
Re-enable many disabled tests

### PR DESCRIPTION
**Pull request by cobbal**
refs: MBL-14474
affects: student, teacher
release note: fix some accessibility strings in discussions

see https://docs.google.com/spreadsheets/d/1OcW3pKgqY_pJfeNeTklyJAcJFcXp3QNR8mwHL8fh0Xg

[run-nightly]

### CODE REVIEW

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/pt6mu3h) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fpt6mu3h)
teacher: [Link](https://rebrand.ly/u25yt07) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fu25yt07)
parent: [Link](https://rebrand.ly/7eg9qmm) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F7eg9qmm)

**Comment by inst-danger** [comment]
Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.32% | 93.32% | 0%

[MBL-14474](https://instructure.atlassian.net/browse/MBL-14474)

#### Affected Apps: Student, Teacher

#### Release Note: 
fix some a11y strings in discussions
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/CoreTests/RichContentEditor/RichContentEditorViewControllerTests.swift [hunk 1]
```diff
@@ -141,16 +138,14 @@ class RichContentEditorViewControllerTests: CoreTestCase, RichContentEditorDeleg
...
-        let files: [File] = databaseClient.fetch()
-        XCTAssertTrue(files.isEmpty)
+        let files: () -> [File] = { self.databaseClient.fetch() }
+        XCTAssertTrue(files().isEmpty)
 
         controller.webView.evaluateJavaScript("document.querySelector('.retry-upload').onclick()")
-        wait(for: [expectation(for: .all, evaluatedWith: self) { () -> Bool in
-            !(self.databaseClient.fetch() as [File]).isEmpty
-        }, ], timeout: 5)
+        waitUntil(30) { print(files()); return !files().isEmpty }
```
Comment (vanwagonet) [inline Core/CoreTests/RichContentEditor/RichContentEditorViewControllerTests.swift:148]: do we want this print to be there?

File: rn/Teacher/ios/TeacherTests/Attendance/RollCallSessionTests.swift [hunk 2]
```diff
@@ -71,7 +71,7 @@ class RollCallSessionTests: TeacherTestCase, RollCallSessionDelegate {
     }
 
     // FIXME: This fails in bitrise, but succeeds locally.
```
Comment (vanwagonet) [inline rn/Teacher/ios/TeacherTests/Attendance/RollCallSessionTests.swift:73]: Should we remove this FIXME then?

**Comment by cobbal** [comment]
@vanwagonet testLinks and testProfileLogsOut were existing, enabled flaky tests. I think I'll disable them. Both of the profile tests have the commonality of failing on Profile.open. From a comment in that file: "opening and closing profile causes flakiness" so I'll probably disable all 3.

Then, I'll trigger nightly another couple times to see if any more flakiness emerges.

**Pull request merged**